index.html

```diff
@@ -0,0 +1,291 @@
+                            <li>javascript</li>
+                        </ul>
+                    </div>
+                    <a href="">
+                        <div class="project">See Project</div>
+                    </a>
+                </div>
+                <img class="img" src="image/card-2.svg" alt="" />
+            </div>
+
+            <div class="card">
+                <img class="img" src="image/card-3.svg" alt="" />
+                <div class="content">
+                    <h2 class="topic">Tonic</h2>
+                    <div class="sub-topic">
+                        <span>CANOPY</span>
+                        <span class="dot"></span>
+
+                        <span>Back End Dev</span>
+                        <span class="dot"></span>
+
+                        <span>2015</span>
+                    </div>
+                    <p class="description">
+                        A daily selection of privately personalized reads; no accounts
+                        or sign-ups required.
+                    </p>
+                    <div class="button-links">
+                        <ul>
+                            <li>html</li>
+                            <li>css</li>
+                            <li>javascript</li>
+                        </ul>
+                    </div>
+                    <a href="">
+                        <div class="project">See Project</div>
+                    </a>
+                </div>
+            </div>
+
+            <div class="card">
+                <div class="content">
+                    <h2 class="topic">
+                        Multi-Post
+                        <br />
+                        Stories
+                    </h2>
+                    <div class="sub-topic">
+                        <span>CANOPY</span>
+                        <span class="dot"></span>
+
+                        <span>Back End Dev</span>
+                        <span class="dot"></span>
+
+                        <span>2015</span>
+                    </div>
+                    <p class="description">
+                        A daily selection of privately personalized reads; no accounts
+                        or sign-ups required.
+                    </p>
+                    <div class="button-links">
+                        <ul>
+                            <li>html</li>
+                            <li>css</li>
+                            <li>javascript</li>
+                        </ul>
+                    </div>
+                    <a href="">
+                        <div class="project">See Project</div>
+                    </a>
+                </div>
+                <img class="img" src="image/card-4.svg" alt="" />
+            </div>
+        </div>
+    </section>
+    <section class="aboutMee">
+        <div id="about" class="myself">
+            <div class="left">
+                <p class="description">
+                <p class="topic">About <br /> Myself</p>
```

Comment:
[OPTIONAL]

- [ ] KIndly wrap your About Myself in a heading tag to make it semantic for accessibility purposes 👍🏼 

e.g  `<h2 class="topic">About <br /> Myself</h2>`